src/app/containers/footer/footer.component.ts

```diff
@@ -0,0 +1,30 @@
+import { Component } from '@angular/core';
+import { Helper, Navigation } from './footer.model';
+
+@Component({
+    selector: 'exui-app-footer',
+    templateUrl: './footer.component.html',
+    styleUrls: ['./footer.component.scss']
+})
+export class FooterComponent {
+    public helpData: Helper = {
+        heading: 'Help',
+        email: {
+            address: 'service-desk@hmcts.gov.uk',
+            text: 'service-desk@hmcts.gov.uk'
+        },
+        phone: {
+            text: '0207 633 4140'
+        },
+        opening: {
+            text: 'Monday to Friday, 8am to 6pm (excluding public holidays)'
+        }
+    };
+    public navigationData: Navigation = {
+        items: [
```

Comment:
Should we have something in place for configuration data so it's not inline with code?

Reply:
This is now resolved. Footer content exported.